chainer/optimizer.py

```diff
@@ -853,3 +853,58 @@ def __call__(self, rule, param):
         xp = cuda.get_array_module(grad)
         with cuda.get_device_from_array(grad):
             xp.clip(grad, self.lower_bound, self.upper_bound, out=grad)
+
+
+class GradientLARS(object):
+
+    """Optimizer/UpdateRule hook function for layer wise adaptive rate scaling.
+
+    See: `Large Batch Training of Convolutional Networks \
+          <https://arxiv.org/abs/1708.03888>`_
+
+    This hook function scales all gradient arrays to fit to the weight norm.
+
+    Args:
+        threashold (float): If weight norm is more than threshold,
+            this function scales all gradient arrays to fit weight norm.
+        weightdecay (float): Coefficient for the weight decay.
```

Comment:
The WeightDecay hook treat the word "weightdecay" as a concatenation of two words. To align with the convention, could you write as `weight_decay`?

Reply:
OK.